pom.xml

```diff
@@ -120,7 +120,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.0.1</version>
+                <version>2.9.1</version>
```

Comment:
Same as above

Reply:
The documentation format for the sns-extended-client works best with javadoc v2.9.1 which was being used for generating the documentation for the prior 1.x versions of the extended client. Locally I was developing on Maven3 which had compatablity issues with v2.9.1 so I had upgraded the version but with maven2 which has been used before to deploy the project before v2.9.1 is compatible.

If it needs to be upped again pls inform.